dotnet/src/SemanticKernel.Skills/Skills.Web/WebFileDownloadSkill.cs

```diff
@@ -69,14 +69,14 @@ public async Task DownloadToFileAsync(string source, SKContext context)
         }
 
         this._logger.LogDebug("Sending GET request for {0}", source);
-        HttpResponseMessage response = await this._httpClient.GetAsync(new Uri(source), context.CancellationToken);
+        using HttpResponseMessage response = await this._httpClient.GetAsync(new Uri(source), HttpCompletionOption.ResponseHeadersRead, context.CancellationToken);
         response.EnsureSuccessStatusCode();
         this._logger.LogDebug("Response received: {0}", response.StatusCode);
 
         using Stream webStream = await response.Content.ReadAsStreamAsync();
         using FileStream outputFileStream = new(Environment.ExpandEnvironmentVariables(filePath), FileMode.Create);
 
-        await webStream.CopyToAsync(outputFileStream, (int)webStream.Length, cancellationToken: context.CancellationToken);
+        await webStream.CopyToAsync(outputFileStream, 81920, cancellationToken: context.CancellationToken);
```

Comment:
Can you please add a comment or set a constant explaining this magic number? I found an explainer here, so I'm not entirely in the dark:
https://learn.microsoft.com/en-us/windows/uwp/debug-test-perf/optimize-file-access

Reply:
Sure. It's the same constant value used by Stream when a size isn't supplied. Unfortunately when targeting netstandard2.0, in order to supply a cancellation token you also need to provide the size... that's not needed when targeting net core, where there's an overload that takes just a token.